android/java/org/chromium/chrome/browser/playlist/kotlin/adapter/recyclerview/PlaylistAdapter.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package org.chromium.chrome.browser.playlist.kotlin.adapter.recyclerview
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.widget.AppCompatImageView
+import androidx.appcompat.widget.AppCompatTextView
+
+import com.bumptech.glide.Glide
```

Comment:
@deeppandya please note that we should not use Glide in new code 
https://github.com/brave/reviews/issues/1391
cc @bridiver

Reply:
I am going to update the in the follow up issue. here we using glide to render already cached image.

Reply:
@stoletheminerals @bridiver Created an issue to remove glide changes for playlist : https://github.com/brave/brave-browser/issues/42448
Also have the draft PR here : https://github.com/brave/brave-core/pull/26690